api/openapi/websocket.yml

```diff
@@ -4,7 +4,7 @@ info:
   description: WebSocket API for sending messages through communication channels.
```

Comment:
This is an async api. I would sugggest we move it to `api/asyncapi`. You can use this [reference](https://www.asyncapi.com/) to help you rewrite it